crates/contract/src/lib.rs

```diff
@@ -434,8 +435,11 @@ impl MpcContract {
         response: SignatureResponse,
     ) -> Result<(), Error> {
         let signer = env::signer_account_id();
+
         log!("respond: signer={}, request={:?}", &signer, &request);
 
+        self.tee_state.assert_caller_is_attested_node();
```

Comment:
Doesn't this only assert that the node has a valid attestation? I don't think it enforces the node to be a participant in the current epoch.